com.unity.netcode.gameobjects/Runtime/Configuration/NetworkConfig.cs

```diff
@@ -53,9 +53,18 @@ public class NetworkConfig
         public uint TickRate = 30;
 
         /// <summary>
-        /// The amount of seconds to wait for handshake to complete before timing out a client
+        /// The amount of seconds for the server to wait for the connection approval handshake to complete before the client is disconnected.
+        ///
```

Comment:
```suggestion
        /// This could happen if for example you're calling external services to authenticate your users and those services timeout.
```
For users that never worked on connected games, this could help.

Reply:
This isn't the appropriate type of comment to add to a Summary section. I'll add some similar flavor to Remarks